docs/content/en/docs/release-notes/_index.md

```diff
@@ -0,0 +1,95 @@
+
+---
+title: "K8ssandra release notes"
+linkTitle: "Release notes"
+weight: 1
+description: Release notes for the open-source K8ssandra community project.
+---
+
+K8ssandra provides a production-ready platform for running Apache Cassandra&reg; on Kubernetes. This includes automation for operational tasks such as repairs, backup and restores, and monitoring. 
+
+Also deployed is Stargate, an open source data gateway that lets you interact programmatically with your Kubernetes-hosted Cassandra resources via a well-defined API. 
+
+The **K8ssandra 1.1.0** release implements a number of key changes, enhancements, and bug fixes, as noted below. For the complete list, see [CHANGELOG-110.md](https://github.com/k8ssandra/k8ssandra/blob/main/CHANGELOG.md).  
+
+**Release date:** 09-April-2021.
+
+## Prerequisites
+
+* A Kubernetes v1.15 or later environment - local or via a supported cloud provider
+* [Helm](https://helm.sh/) v3
+
+## Supported Kubernetes environments
+
+* Open-source [kubernetes.io](https://kubernetes.io)
+* [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) (GKE)
+* [Microsoft Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/) (AKS)
+* [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS)
+* [MiniKube](https://minikube.sigs.k8s.io/docs/)
+* [Kind](https://kind.sigs.k8s.io/)
+* [K3D](https://k3d.io/)
+
+## K8ssandra deployed components
+
+The K8ssandra helm chart deploys the following components. Some are optional, and depending on the configuration, may not be deployed:
```

Comment:
I guess we'll also want to update the FAQ that includes our component versions as well:

https://k8ssandra.io/docs/faqs/#what-components-does-k8ssandra-install

Do you want to hit that as well @johnsmartco once we nail down the versions to mention here in this PR?

Reply:
To avoid having to keep duplicate content in sync, I'll have the components/versions in the latest RNs, and change the FAQ to just link to that RNs #section.

Reply:
Updated FAQ now points to components/versions list in RNs.